go.mod

```diff
@@ -3,12 +3,9 @@ module github.com/keptn/go-utils
 go 1.16
 
 require (
-	github.com/cloudevents/sdk-go/v2 v2.3.1
-	github.com/go-openapi/errors v0.19.2
-	github.com/go-openapi/strfmt v0.19.3
-	github.com/go-openapi/swag v0.19.14
-	github.com/go-openapi/validate v0.19.4
+	github.com/cloudevents/sdk-go/v2 v2.4.1
+	github.com/google/go-cmp v0.5.2 // indirect
 	github.com/google/uuid v1.2.0
 	github.com/stretchr/testify v1.7.0
-	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
```

Comment:
as part of dependency cleanup, can we also strip down the number of yaml parsing libraries down to one. I think we are using multiple different libs for that for no reason in keptn/keptn. which one do you pefer?

Reply:
i'd say let's go with `gopkg.in/yaml.v3`